src/http/request.ts

```diff
@@ -40,9 +42,11 @@ export class DrashRequest extends Request {
   constructor(
     originalRequest: Request,
     pathParams: Map<string, string>,
+    connInfo: ConnInfo,
   ) {
     super(originalRequest);
     this.#path_params = pathParams;
+    this.connInfo = connInfo;
```

Comment:
we going with camel case?

Reply:
nah should change it to snake tbh, one mo

Reply:
https://github.com/drashland/drash/pull/569/commits/dd5a457c4632285e6d1263cd4b852b363e0fa0e0